tests/dind-auto-install/Earthfile

```diff
@@ -2,8 +2,8 @@ VERSION 0.6
 
 all:
     BUILD +test \
-        # Pin the version here as 20.10.15 seems to have introduced a bug.
         --BASE_IMAGE=docker:20.10.14-dind \
+        --BASE_IMAGE=docker:20.10.15-dind \
```

Comment:
It might be good for us to catch any future incompatibility. I would maybe keep the unpinned version here.